internal/services/fluidrelay/fluid_relay_servers_resource.go

```diff
@@ -0,0 +1,270 @@
+package fluidrelay
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/identity"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/fluidrelay/2022-05-26/fluidrelayservers"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/fluidrelay/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ServerModel struct {
+	Name             string                                     `tfschema:"name"`
+	ResourceGroup    string                                     `tfschema:"resource_group_name"`
+	Location         string                                     `tfschema:"location"`
+	StorageSKU       string                                     `tfschema:"storage_sku"`
+	FrsTenantId      string                                     `tfschema:"frs_tenant_id"`
+	OrdererEndpoints []string                                   `tfschema:"orderer_endpoints"`
+	StorageEndpoints []string                                   `tfschema:"storage_endpoints"`
+	Tags             map[string]string                          `tfschema:"tags"`
+	Identity         []identity.ModelSystemAssignedUserAssigned `tfschema:"identity"`
+}
+
+func (s *ServerModel) flattenIdentity(input *identity.SystemAndUserAssignedMap) error {
+	if input == nil {
+		return nil
+	}
+	config := identity.SystemAndUserAssignedMap{
+		Type:        input.Type,
+		PrincipalId: input.PrincipalId,
+		TenantId:    input.TenantId,
+		IdentityIds: make(map[string]identity.UserAssignedIdentityDetails),
+	}
+	for k, v := range input.IdentityIds {
+		config.IdentityIds[k] = identity.UserAssignedIdentityDetails{
+			ClientId:    v.ClientId,
+			PrincipalId: v.PrincipalId,
+		}
+	}
+	model, err := identity.FlattenSystemAndUserAssignedMapToModel(&config)
+	if err != nil {
+		return err
+	}
+	s.Identity = *model
+	return nil
+}
+
+type Server struct{}
+
+var _ sdk.ResourceWithUpdate = (*Server)(nil)
+
+func (s Server) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.FluidRelayServerName,
+		},
+		"resource_group_name": commonschema.ResourceGroupName(),
+		"location":            commonschema.Location(),
+		"tags":                commonschema.Tags(),
+		"identity":            commonschema.SystemAssignedUserAssignedIdentityOptional(),
+		"storage_sku": {
```

Comment:
`storage_sku` doesn't get returned from the api. Is there a reason why it's not returned? It looks like a bug and we should open a ticket on that repo

Reply:
This should also be `ForceNew`